features/dump.feature

```diff
@@ -32,3 +37,22 @@ Feature: The dump command
       Util\.system! scp -C user@www\.example\.com:~\/dumps\/dump_for_download.dump .*?\/tmp\/aruba\/tmp\/staging.dump
       """
       And the output should contain "> Dumped the staging database to tmp/staging.dump"
+
+  Scenario: Creating a dump of one of multiple remote databases and loading it locally
+    Given a file named "Capfile" with "Capfile exists"
+    And a file named "config/deploy.rb" with:
+    """
+    """
```

Comment:
Please also have this as `a file named "config/deploy.rb" with "deploy.rb exists"`.